apps/mdn/mdn-aws/infra/multi_region/tf/main.tf

```diff
@@ -105,19 +105,20 @@ module "memcached-dev" {
 # MySQL
 #########################################
 
-#module "mysql-stage" {
-#    source = "rds"
-#    # DBName must begin with a letter and contain only alphanumeric characters
-#    mysql_db_name = "mdnstage"
-#    mysql_username = "mdn"
-#    mysql_password = "${var.mysql_stage_password}"
-#    mysql_identifier = "mdnstage"
-#    # stage instace class is much smaller than prod
-#    mysql_instance_class = "db.t2.medium"
-#    mysql_backup_retention_days = 0
-#    vpc_id = "${var.vpc_id}"
-#}
-#
+module "mysql-stage" {
+    source = "rds"
+    # DBName must begin with a letter and contain only alphanumeric characters
+    mysql_env     = "stage"
+    mysql_db_name = "mdnstage"
```

Comment:
I think `mysql_db_name` is used to create a database of that name within the mysql instance? Do we have to change this to `developer_allizom_org` to stay consistent with a dump of our current stage db (or a dump of the prod db that has been anonymized)?

Reply:
+1, I'll update soonish.